src/unittests/f32_interval.spec.ts

```diff
@@ -3425,6 +3426,122 @@ g.test('modfInterval')
     );
   });
 
+interface MatrixToScalarCase {
+  input: number[][];
+  expected: number | IntervalBounds;
+}
+
+g.test('determinantInterval')
+  .paramsSubcasesOnly<MatrixToScalarCase>([
+    // Exterme values, i.e. subnormals, very large magnitudes, and those lead to
+    // non-precise products, are intentionally not tested, since the accuracy of
+    // determinant is restricted to well behaving inputs. Handling all cases
+    // requires ~23! options to be calculated in the 4x4 case, so is not
+    // feasible.
+    {
+      input: [
+        [1, 2],
+        [3, 4],
+      ],
+      expected: -2,
+    },
+    {
+      input: [
+        [-1, 2],
+        [-3, 4],
+      ],
+      expected: 2,
+    },
+    {
+      input: [
+        [11, 22],
+        [33, 44],
+      ],
+      expected: -242,
+    },
+    {
+      input: [
+        [5, 6],
+        [8, 9],
+      ],
+      expected: -3,
+    },
+    {
+      input: [
+        [4, 6],
+        [7, 9],
+      ],
+      expected: -6,
+    },
+    {
+      input: [
+        [4, 5],
+        [7, 8],
+      ],
+      expected: -3,
+    },
+    {
+      input: [
+        [1, 2, 3],
+        [4, 5, 6],
+        [7, 8, 9],
+      ],
+      expected: 0,
```

Comment:
Surprising, but correct.

I found an online tool for this from Wolfram Alpha.
https://www.wolframalpha.com/input?i2d=true&i=determinant%7B%7B1%2C2%2C3%7D%2C%7B4%2C5%2C6%7D%2C%7B7%2C8%2C9%7D%7D